internal/backend/post.go

```diff
@@ -6,20 +6,24 @@ import (
 	"io"
 	"time"
 
-	"github.com/nedrocks/delphisbe/internal/datastore"
+	"go.uber.org/multierr"
 
 	"github.com/nedrocks/delphisbe/graph/model"
+	"github.com/nedrocks/delphisbe/internal/datastore"
 	"github.com/nedrocks/delphisbe/internal/util"
 	"github.com/sirupsen/logrus"
 )
 
 func (d *delphisBackend) CreatePost(ctx context.Context, discussionID string, participantID string, input model.PostContentInput) (*model.Post, error) {
```

Comment:
Not sure where this should be but we need to analyze the structure of the incoming PostContentInput to ensure the fields are as expected. For instance we should make sure the string parses and that the number of tokens match the number of items in the array for mentioned entitites, etc. Probably in this function?

Reply:
updated